### DISCUSSION TITLE
React Server Components

### DISCUSSION THREAD

**Discussion by marcfielding1** [vercel/next.js#20878] [Help]
Because I'm impatient and super excited - where can we track progress on React Server Components, please? We're currently pre-rendering 150,000 product pages with some search functionality - I think Server Components present us with some very interesting options for keeping out sitespeed up.

I know it's just been announced so it's going to hard to give anything concrete right now but we'd love to stay up to date.

**Comment by leerob** [comment]
You can try out and view the code for [Next.js + React Server Components here](https://github.com/vercel/next-server-components).

There is no official timeline on when Server Components will land in Next.js. You can track progress by following PRs/commits 😄
